livekit-agents/livekit/agents/voice/agent_session.py

```diff
@@ -614,12 +614,12 @@ def clear_user_turn(self) -> None:
 
         self._activity.clear_user_turn()
 
-    def commit_user_turn(self) -> None:
+    def commit_user_turn(self, *, final_transcript_timeout: float = 2.0) -> None:
```

Comment:
IMO final isn't needed
```suggestion
    def commit_user_turn(self, *, transcript_timeout: float = 2.0) -> None:
```